src/core/Form/FilterInput/FilterInput.test.tsx

```diff
@@ -0,0 +1,199 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { axeTest } from '../../../utils/test/axe';
+
+import { FilterInput } from './FilterInput';
+
+interface Product {
+  name: string;
+  price: number;
+  tax: boolean;
+}
+
+const tools: Product[] = [
+  { name: 'Jackhammer', price: 230, tax: false },
+  { name: 'Hammer', price: 15, tax: true },
+  { name: 'Sledgehammer', price: 36, tax: false },
+  { name: 'Spade', price: 50, tax: true },
+  { name: 'Powersaw', price: 150, tax: false },
+];
+
+const filter = (tool: Product, query: string) => tool.name.includes(query);
+
+const TestFilterInput = (
+  <FilterInput
+    labelText="Label"
+    items={tools}
+    onFiltering={(filtered) => console.log(filtered)}
```

Comment:
Console logs might introduce interesting side-effects when run with CI, wouldn't use here if not absolutely necessary. Rather use a spy or mock etc?

Reply:
https://jestjs.io/docs/en/mock-function-api#mockfnmockresults

Reply:
Maybe this could or something similar could be used to test the filter functionality as well?

Reply:
`console.log`s are now removed and using mock functions. Also using empty array for `items`-prop and mock function for the `filterFunc`.

There is now test for the `onFilter` prop. It basically tests also the `items` and `filterFunc` props as it can not work without them really.